data_steward/spec/pages/file_transfer_procedures.md

```diff
@@ -1,43 +1,98 @@
-1. Create a folder inside the bucket to load the data into.
-Suggested folder naming scheme would be the date of the upload.
-1. In your desktop environment, navigate to the local folder where you've downloaded the test data set.
-1. Drag and drop the test data set files from your local folder into the browser window where the Google Cloud Storage Browser is open to the folder created in step 3.
+   * Name of Organization that will perform the upload
+   * Composition of data upload: Will data be uploaded for only one Organization or for multiple Organizations? If for more than one Organization, also include the names of all Organizations whose data will be uploaded.
+   * Name, institutional email address, and pmi-ops.org account information of the Data Steward(s)
 
-## Review the Results
+Within five business days you will receive a response from the sysadmin containing:
 
-Within 6 hours of uploading the data set, the report page should be updated with the status of your upload. A more detailed report should appear in your GCS bucket if any issues were detected. 
 
-## Report Issues
+   * The name of your Organization’s assigned Google Cloud Storage Bucket
+   * Direct link to aforementioned Bucket
 
-If you experience any issues in this process, please report an issue on our [JIRA site](https://precisionmedicineinitiative.atlassian.net).
+__Note:__ You can concurrently send the pmi-ops.org account and the Bucket creation requests to sysadmin@pmi-ops.org. However, the request emails need to be separate for tracking purposes.
+
+---
+
+## DE-PROVISIONING
+__Step 1:__  Request account to be de-provisioned by sending an email to sysadmin@pmi-ops.org with the subject line “Account De-Provision”.  Include the following information:
+  * Name of Organization or Individual to be de-provisioned.
+  * If it is an individual include a replacement contact.
+
+---
+
+## UPLOADING EHR DATA TO THE DRC
+This part of the process steps through the procedure for uploading EHR data files to the Data and Research Center (DRC) Google Cloud Storage Buckets.The process applies to all program partner organizations that are responsible for sharing participant data (e.g., EHR) with the DRC.  It details the required documentation and parameters needed as well as the testing required to upload EHR data to the DRC Google Cloud Storage Buckets.
+
+__Step 1:__ Log out of any currently active Google IDs. Log into MyAccount (https://myaccount.google.com/)  with the pmi-ops.org credentials provided by the DRC User Admin.
+
+__Step 2:__ Data Stewards will receive an email from the DRC User Admin containing a direct link to the assigned Bucket. Open a web browser and navigate to the Google Cloud Storage Bucket via the direct link shared with you by the DRC User Admin. 
+
+NOTE: If you receive the error message shown below, please check that the pmi-ops.org account is the active account. 
+
+__PICTURE FROM SOP NEEDS INSERTED HERE__
```

Comment:
@cmcphilamy send me the pictures and I can add them for you. Please include step number in the file names so I know where to put them.

Reply:
One
![one](https://user-images.githubusercontent.com/31285041/32297630-b6020734-bf1d-11e7-84aa-cb06a386e515.png)
Two
![two](https://user-images.githubusercontent.com/31285041/32297627-b5e892ea-bf1d-11e7-97e6-dc8bbe849b03.png)
Three
![three](https://user-images.githubusercontent.com/31285041/32297629-b5f51790-bf1d-11e7-8d4d-cb8225be7267.png)